eng/tools/GenerateFiles/Directory.Build.props.in

```diff
@@ -1,7 +1,6 @@
 <Project>
   <PropertyGroup>
     <DefaultNetCoreTargetFramework>${DefaultNetCoreTargetFramework}</DefaultNetCoreTargetFramework>
-    <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
```

Comment:
We don't have any projects lacking a `$(TargetFramework)` setting and this can cause problems where we use `$(TargetFrameworks)`